src/minimize_lbfgs.jl

```diff
@@ -0,0 +1,506 @@
+using LinearAlgebra
```

Comment:
Please collect using-statements in `Manifolds.jl` main file (I think we already have LinearAlgebra in there) – and for best of cases state the functions you need explicitly, to keep the “inner name space” of Manifolds.jl not too crowded.

Reply:
removed